features/README.md

```diff
@@ -7,5 +7,4 @@
 |     2     | [Cache Coherency](./cache-coherency)                      | Under Development |
 
 
-Features are a second type of extension that __ONLY__ change behavior of the CPU without adding instructions, control registers, or any other similar aspect. They generally mandate a specific behavior when certain actions occur but do __NOT__
-change how instructions are decoded.
+Features are a second type of extension that __ONLY__ change behavior of the CPU without adding instructions, control registers, or any other similar aspect. They generally only mandate a specific behavior when certain actions occur.
```

Comment:
```suggestion
Features are a second type of extension that __ONLY__ change behavior of the CPU without adding instructions, control registers, or any other similar aspect. They generally only mandate a specific behavior when certain otherwise unspecified actions occur.
```
clarity?